src/bindings/jna/CMakeLists.txt

```diff
@@ -12,6 +12,12 @@ if (Java_JAVAC_EXECUTABLE)
 	# as 9.x.x > 1.8 and 1.8.x > 1.8 and 1.9.x > 1.8
 	# ~~~
 	if ((${Java_VERSION} VERSION_GREATER "1.8.0") OR (${Java_VERSION} VERSION_EQUAL "1.8.0"))
+		if (NOT BUILD_SHARED)
```

Comment:
Which problem was the reason to add this? Is maybe ONLY_SHARED the better option here?

Reply:
This change was introduced, because I disabled the unit test in the PR, and did not insert `ADD_TEST` at the same location after I re-enabled the test.

Reply:
> Which problem was the reason to add this?

The unit test for the [binding fails](https://travis-ci.org/sanssecours/elektra/jobs/445840045) (with a `NoClassDefFoundError`) on Ubuntu, if we do not enable `BUILD_SHARED`.

> Is maybe ONLY_SHARED the better option here?

Yep, that should work too. I will check, if that is a valid option.

Reply:
Looks like [using `ONLY_SHARED` does not fix the problem](https://travis-ci.org/ElektraInitiative/libelektra/builds/447434672?utm_source=github_status&utm_medium=notification).